workers/ohsome_quality_analyst/indicators/currentness/indicator.py

```diff
@@ -105,18 +107,12 @@ def calculate(self) -> None:
                 break
         median_diff = int(self.result.timestamp_oqt.year) - int(median_year)
```

Comment:
can you add a comment above what you will do in the next lines. without context it just looks confusing. I guess that you are deriving the median year, which is the timestamp for which 50% of the OSM data has been edited for the last time. But what is `median_diff` and why are you dividing by 10?

Just add a bit more explanations here so that we understand also in 6 months how this indicator works. :)

Reply:
Done